### PR TITLE
CHF-240 : Adding readme.md files for below devices

### PR DESCRIPTION
**Pull request by ShunmugaSundar**
1. SmartSense Moisture Sensor
   1. SmartSense Temp/Humidity Sensor
   2. SmartSense Multi Sensor
   3. SmartSense Open/closed Sensor
   4. SmartPower Outlet
   5. Connected Cree Bulb
   6. Aeon Multisensor
   7. Aeon Multisensor Gen5
   8. Aeon Multisensor 6
   9. SmartSense Motion Sensor

### CODE REVIEW

**Comment by ShunmugaSundar** [comment]
@jackchi Please review the changes and merge

@parijatdas

**Comment by jackchi** [comment]
We're removing support for all the Aeon Multi Sensors for now, so no need for the documentation on those.

File: devicetypes/smartthings/cree-bulb.src/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* **Polling** - represents that poll() can be implemented for the device
+* **Refresh** - _refresh()_ command for status updates
+* **Switch** - can detect state (possible values: on/off)
+* **Switch Level** - represents current light level, usually 0-100 in percent
+
+## Device Health
+
+A Category C6 Connected Cree LED Bulb that has 20min check-in interval
+
+## Battery Specification
```
Comment (jackchi) [inline devicetypes/smartthings/cree-bulb.src/README.md]: For powered devices no need for battery info.

File: devicetypes/smartthings/cree-bulb.src/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* [Battery]($battery)
+
+## Capabilities
+
+* **Actuator** - represents that a Device has commands
+* **Configuration** - _configure()_ command called when device is installed or device preferences updated
+* **Polling** - represents that poll() can be implemented for the device
+* **Refresh** - _refresh()_ command for status updates
+* **Switch** - can detect state (possible values: on/off)
+* **Switch Level** - represents current light level, usually 0-100 in percent
```
Comment (jackchi) [inline devicetypes/smartthings/cree-bulb.src/README.md:21]: Health Check? I just merged a PR, pull Master again.

File: devicetypes/smartthings/smartpower-outlet.src/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+* **Refresh** - _refresh()_ command for status updates
+* **Power Meter** - detects power meter for device in either w or kw.
+* **Health Check** - indicates ability to get device health notifications
+* **Sensor** - detects sensor events
+
+## Device Health
+
+A Category C1 smart power outlet that has 20min check-in interval
+
+## Battery Specification
```
Comment (jackchi) [inline devicetypes/smartthings/smartpower-outlet.src/README.md]: Same as Cree here.

File: devicetypes/smartthings/smartsense-moisture-sensor.src/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* **Configuration** - _configure()_ command called when device is installed or device preferences updated
+* **Battery** - defines device uses a battery
+* **Refresh** - _refresh()_ command for status updates
+* **Temperature Measurement** - defines device measures current temperature
+* **Water Sensor** - can detect presence of water (dry or wet)
+* **Health Check** - indicates ability to get device health notifications
+
+## Device Health
+
+A Category C2 moisture sensor that has 120min check-in interval
```
Comment (jackchi) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/README.md]: This value `7200` might be misleading; perhaps we can put how often it periodically reports back and how we decide to 2x it.

File: devicetypes/smartthings/smartpower-outlet.src/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+
+Works with: 
+
+* [Samsung SmartPower Outlet](https://shop.smartthings.com/#!/products/smartpower-outlet)
+
+## Table of contents
+
+* [Capabilities](#capabilities)
+* [Health]($health)
```
Comment (jackchi) [inline devicetypes/smartthings/smartpower-outlet.src/README.md]: These `Health` & `Battery` URLs gives 404. It needs to be `Device Health` or the Table of Content needs to change.

**Comment by skt123** [comment]
@jackchi 
We have incorporated the review comments. Please review and merge.

@ShunmugaSundar @parijatdas

**Comment by jackchi** [comment]
👍

**Pull request merged**